src/nwb_benchmarks/benchmarks/netbench_streaming.py

```diff
@@ -84,30 +83,46 @@ def test_case(self):
     def track_bytes_downloaded(self, cache):
         return cache["bytes_downloaded"]
 
+    track_bytes_downloaded.unit = "bytes"
```

Comment:
BTW it looks like this is the way to associate `unit` to a specific `track_` function instead of the benchmark itself

Reply:
Nice! I was digging around the docs, but I did not find this part about unit, so that's great.